test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/TagHelpersWebSite.Employee.Create.Invalid.html

```diff
@@ -17,8 +17,8 @@
             <hr />
             <div class="text-danger validation-summary-errors" data-valmsg-summary="true">
             <ul><li>The field Age must be between 10 and 100.</li>
-<li>The value &#x27;z&#x27; is not valid for Salary.</li>
 <li>The JoinDate field is required.</li>
+<li>The value &#x27;z&#x27; is not valid for Salary.</li>
```

Comment:
Can we use ordering `[Display(...)]` on the model properties here so that we don't have this fragility?